node/primitives/src/lib.rs

```diff
@@ -284,3 +285,28 @@ impl View {
 		self.0.contains(hash)
 	}
 }
+
+/// The output of a collator.
+#[derive(Clone, Encode, Decode)]
+pub struct Collation {
```

Comment:
Associated guide update?

Reply:
Oh, I see the other comment. Disregard :) 

https://github.com/paritytech/polkadot/pull/1557#discussion_r469249658